hook/iptables.go

```
@@ -109,6 +109,10 @@ func generateIptables(
 		return fmt.Errorf("could not retrieve container brigde interface")
 	}
 
+	preroutingChain := viper.GetString("iptables.chains.prerouting")
```

What is the behavior of this when `iptables.chains.prerouting` is not defined in conf ? I believe that viper return empty string so it will be iptable which will fail.

---

Yep, iptables command will fail. We can handle this properly and just return a pretty error in the task failure message if prefered.

---

We agree that executor is not responsible of creating iptables chains. So why not failing early if no chains are configured ?

---

I agree with tgermain. This is the behavior implemented in the acl hook.

---

Done